drivers/serial/uart_mcux.c

```diff
@@ -40,6 +42,8 @@ static int uart_mcux_configure(const struct device *dev,
 	uint32_t clock_freq;
 	status_t retval;
 
+	pinctrl_state_set(config->pincfg, PINCTRL_STATE_DEFAULT);
```

Comment:
If error checking is not required, better make it _explicit_, e.g. `(void)pinctrl_state_set(...)`.

Reply:
Added error checking.